src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -232,6 +233,9 @@ public without sharing class UTIL_OrgTelemetry_SVC {
     private void handleNPSPSettings() {
         STG_SettingsService stgSvc = new STG_SettingsService();
         Addr_Verification_Settings__c addrVerifSettings = UTIL_CustomSettingsFacade.getAddressVerificationSettings();
+        if (Test.isRunningTest()) {
```

Comment:
We can add the configuration in the unit test instead using `Test.isRunningTest()` in the code here.

Reply:
Moved configuration as suggested and eliminated unnecessary Test.isRunningTest()